source/common/quic/platform/quiche_flags_constants.h

```diff
@@ -0,0 +1,44 @@
+                 true) /* Envoy only supports RFC-v1 in the long term, so disable IETF draft 29    \
+                          implementation by default. */                                            \
+  KEY_VALUE_PAIR(                                                                                  \
+      quic_default_to_bbr,                                                                         \
+      true) /* This flag enables BBR, otherwise QUIC will use Cubic which is less performant */
+
+#define OVERRIDDEN_PROTOCOL_FLAGS(KEY_VALUE_PAIR)                                                  \
+  KEY_VALUE_PAIR(                                                                                  \
+      quic_header_size_limit_includes_overhead,                                                    \
+      false) /* Do not include 32-byte per-entry overhead while counting header size. */           \
+  KEY_VALUE_PAIR(                                                                                  \
+      quic_buffered_data_threshold,                                                                \
+      2 * ::Envoy::Http2::Utility::OptionsLimits::                                                 \
+              DEFAULT_INITIAL_STREAM_WINDOW_SIZE) /* Set send buffer twice of max flow control     \
+                                                     window to ensure that stream send buffer      \
+                                                     always takes all the data. The max amount of  \
+                                                     data buffered is the per-stream high          \
+                                                     watermark + the max flow control window of    \
+                                                     upstream. The per-stream high watermark       \
+                                                     should be smaller than max flow control       \
+                                                     window to make sure upper stream can be flow  \
+                                                     control blocked early enough not to send more \
+                                                     than the threshold allows.                    \
+                                                     TODO(#8826) Ideally we should use the         \
+                                                     negotiated value from upstream which is not   \
+                                                     accessible for now. 512MB is way to large,    \
+                                                     but the actual bytes buffered should be bound \
+                                                     by the negotiated upstream flow control       \
+                                                     window. */
+
+namespace quiche {
+
+const std::string EnvoyQuicheReloadableFlagPrefix =
+    "envoy.reloadable_features.FLAGS_envoy_quic_reloadable_flag_";
+
```

Comment:
`inline constexpr absl::string_view` to avoid creating a new copy for each compilation unit?

https://abseil.io/tips/140

Reply:
done